packages/frontend/src/pages/EventLogs/components/EventLog.tsx

```diff
@@ -79,16 +78,6 @@ export const EventLog = ({
             title={eventlog.type.description}
             className={eventLogTypeColors[eventlog.type.key]}
           />
-
-          {eventlog.hidden ? (
```

Comment:
Why this deleted?

Reply:
@kristoferlund the frontend was expecting a `hidden` parameter from the `api/event-log`, but only the older version of the api returns this value, the api2 doesn't so I tought it was an legacy behaviour. We should either remove this part of the code in the frontend or add the property on api2

Reply:
Yes, it is kind of an edge case. We didn't want to show quant scores from other quantifiers during an ongoing quantification in the logs. But .. like you say .. the feature has not been brought over to the new api and I don't believe that to be a significant risk anymore anyway. Let's retire it like you did.